### PR TITLE
feat: `onGridUpdated` event(#866)

### PR DESCRIPTION
**Pull request by js87zz**
### Please check if the PR fulfills these requirements
- [x] It's submitted to right branch according to our branching model
- [x] It's right issue type on title
- [x] When resolving a specific issue, it's referenced in the PR's title (e.g. `fix #xxx[,#xxx]`, where "xxx" is the issue number)
- [x] The commit message follows our guidelines
- [x] Tests for the changes have been added (for bug fixes/features)
- [ ] Docs have been added/updated (for bug fixes/features)
- [ ] It does not introduce a breaking change or has description for the breaking change

### Description
* related issue(#866)

---
Thank you for your contribution to TOAST UI product. 🎉 😘 ✨

### CODE REVIEW

File: packages/toast-ui.grid/src/view/root.tsx [hunk 1]
```diff
@@ -23,35 +20,31 @@ export class Root extends Component<Props> {
...
+       * @property {Grid} instance - Current grid instance
+       */
+      eventBus.trigger('onGridMounted', gridEvent);
+    });
   }
 
   public componentWillUnmount() {
-    const { onGridBeforeDestroy, store } = this.props;
+    const { store } = this.props;
+    const eventBus = getEventBus(store.id);
```
Comment (jung-han) [inline packages/toast-ui.grid/src/view/root.tsx]:
```js
const eventBus = getEventBus(this.props.store.id);
```
한 줄로 작성할 수 있겠네요 ㅎㅎ

File: packages/toast-ui.grid/src/view/root.tsx [hunk 2]
```diff
@@ -23,35 +20,31 @@ export class Root extends Component<Props> {
   }
 
   public componentDidMount() {
-    const { onGridMounted, store } = this.props;
+    const { store } = this.props;
+    const eventBus = getEventBus(store.id);
```
Comment (jung-han) [inline packages/toast-ui.grid/src/view/root.tsx]: 여기도 한 줄로 작성할 수 있겠네요

File: packages/toast-ui.grid/src/grid.tsx [hunk 3]
```diff
@@ -34,7 +35,7 @@ import i18n from './i18n';
 import { getText } from './query/clipboard';
 import { getInvalidRows } from './query/validation';
 import { isSupportWindowClipboardData, setClipboardSelection, cls, dataAttr } from './helper/dom';
-import { findPropIndex, isUndefined, mapProp, hasOwnProp } from './helper/common';
+import { findPropIndex, isUndefined, mapProp, hasOwnProp, extract } from './helper/common';
```
Comment (jung-han) [inline packages/toast-ui.grid/src/grid.tsx]: 뭔가 뒤늦은 것 같지만.. `extract`대신 `omit`과 `pick`이 쌍이 되는게 더 자연스러울 것 같습니다.

File: packages/toast-ui.grid/src/grid.tsx [hunk 4]
```diff
@@ -305,16 +307,13 @@ export default class Grid {
...
-        store={store}
-        dispatch={dispatch}
-        rootElement={el}
-        onGridMounted={onGridMounted}
-        onGridBeforeDestroy={onGridBeforeDestroy}
-      />,
-      el
-    );
+    // eslint-disable-next-line prettier/prettier
+    const lifeCycleEvent = extract(options, 'onGridMounted', 'onGridBeforeDestroy','onGridUpdated');
```
Comment (jung-han) [inline packages/toast-ui.grid/src/grid.tsx]: `extract`대신 `pick`로 수정한다면 eslint 주석도 지워질 수 있겠네요

File: packages/toast-ui.grid/src/grid.tsx (hunk 3 above)
Comment (js87zz) [reply packages/toast-ui.grid/src/grid.tsx]: 그러네요ㅋ 타입도 Pick 타입을 쓰고있고 이게 쌍으로 더 좋겠네요~수정하겠습니다!

**Pull request merged**